test/proxy_test.rb

```diff
@@ -62,4 +62,9 @@ def test_augments_proxy_headers
       assert_equal "::1, 10.0.3.1", headers["HTTP_X_FORWARDED_SERVER"]
     end
   end
+
+  def test_empty_header
+    response = Net::HTTP.get(URI("http://empty-header.dev:20557/"))
+    assert_equal "an empty header is tolerated", response
```

Comment:
Maybe check that the header do exists in the hash? It could be tolerated, but not passed along. Something like (I'm not sure of actual objects):

```ruby
assert_equal "", response.headers["Access-Control-Expose-Headers"]
```